cpp/src/parquet/parquet.thrift

```diff
@@ -410,7 +410,7 @@ struct BsonType {
 struct VariantType {
   // The version of the variant specification that the variant was
   // written with.
-  1: optional i8 specification_version
+  1: optional i8 specification_version = 1
```

Comment:
Why do we need this change? https://github.com/apache/parquet-format/blob/master/src/main/thrift/parquet.thrift#L412 does not have this so this change will get removed in any followup sync of this file.

Reply:
Ah. You are right. Let me check that out.